dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/utils/MessageConstants.java

```diff
@@ -34,6 +34,7 @@ public class MessageConstants {
     public static final String OPENAPI_MERGE_INFO_PARSE_ERROR_CWWKO1665W = "OPENAPI_MERGE_INFO_PARSE_ERROR_CWWKO1665W"; //$NON-NLS-1$
     public static final String OPENAPI_MERGE_INVALID_NAME_CWWKO1666W = "OPENAPI_MERGE_INVALID_NAME_CWWKO1666W"; //$NON-NLS-1$
     public static final String OPENAPI_MERGE_UNUSED_INCLUDE_CWWKO1667W = "OPENAPI_MERGE_UNUSED_INCLUDE_CWWKO1667W"; //$NON-NLS-1$
+    public static final String OPENAPI_USING_WRONG_NAME_SOURCE_CWWKO1680W = "OPENAPI_USING_WRONG_NAME_SOURCE_CWWKO1680W";
```

Comment:
There were three messages added but only one message constant. Are the other messages not used yet?